docs/UserGuide.adoc

```diff
@@ -60,19 +65,23 @@ Format: `help`
 === Adding a person: `add` / `a`
 
 Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS b/Birthday(DD/MM/YYYY) [t/TAG]...`
-Shorthand: `a n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS b/Birthday(DD/MM/YYYY) [w/WEBSITE] [t/TAG]...`
```

Comment:
At the very end you need to add a `+` for newline.

Reply:
Our book is Death Note or DeathNote?

Reply:
Missed it by accident. Updated.